docs/sidebars.js

```diff
@@ -7,77 +7,79 @@ const sidebars = {
       label: 'Bolt for JavaScript',
       className: 'sidebar-title',
     },
-    'getting-started',
-    {
-      type: 'category',
-      label: 'Slack API calls',
-      items: ['concepts/message-sending', 'concepts/web-api'],
-    },
-    {
-      type: 'category',
-      label: 'Events',
-      items: ['concepts/message-listening', 'concepts/event-listening'],
-    },
-    {
-      type: 'category',
-      label: 'App UI & Interactivity',
-      items: [
-        'concepts/acknowledge',
-        'concepts/shortcuts',
-        'concepts/commands',
-        'concepts/actions',
-        'concepts/creating-modals',
-        'concepts/updating-pushing-views',
-        'concepts/view-submissions',
-        'concepts/select-menu-options',
-        'concepts/publishing-views',
-      ],
-    },
-    'concepts/assistant',
-    'concepts/custom-steps',
+    { type: 'html', value: '<hr>' },
     {
       type: 'category',
-      label: 'App Configuration',
+      label: 'Guides',
+      collapsed: false,
```

Comment:
and this is new, too. Everything else is github not understanding what's going on. it is the same!